pkg/kubectl/cmd/convert/convert.go

```diff
@@ -263,7 +264,10 @@ func asVersionedObjects(infos []*resource.Info, specifiedOutputVersion schema.Gr
 
 		converted, err := tryConvert(scheme.Scheme, info.Object, targetVersions...)
 		if err != nil {
-			return nil, err
+			//Dont fail on error converting objects.
+			//Simply warn the user with the error returned from api-machinery and continue with the rest of the file
+			fmt.Fprintln(os.Stderr, err.Error())
```

Comment:
I think, type of error matters. Perhaps we can only continue if the error is notfound error?

Reply:
@ardaguclu yep. i think you are right. i will update this PR with that. Thanks